pkg/providers/aws/provider_postgres.go

```diff
@@ -726,7 +727,7 @@ func (p *PostgresProvider) CreateRDSAvailabilityAlert(ctx context.Context, cr *v
 			return errorUtil.Wrap(err, fmt.Sprintf("exception calling Create prometheusrule: %s", alertRuleName))
 		}
 	}
-	p.Logger.Info(fmt.Sprintf("PrometheusRule: %s reconcilced successfully.", pr.Name))
+	p.Logger.Info(fmt.Sprintf("PrometheusRule: %s reconciled successfully.", pr.Name))
```

Comment:
sorry for being pedantic with this, but can we make this all lower case. Just to try an stick with the conventions we have in place already in the operator.